lte/gateway/c/oai/lib/pcef/pcef_handlers.cpp

```diff
@@ -209,31 +210,35 @@ static int get_uli_from_session_req(
     return 0;
   }
 
-  uli[0] = 130;  // TAI and ECGI - defined in 29.061
+  uli[0] = saved_req->uli.present;
```

Comment:
I think this is not a correct change. When both TAI and ECGI are present the value should be 130, if only TAI present the value should be 128, if only ECGI present the value should be 129. With the way the following are defined: 
`#define ULI_TAI 0x08`
`#define ULI_ECGI 0x10`
`uli.present` will have value 24.